piker/data/feed.py

```diff
@@ -86,14 +106,28 @@ class _FeedsBus(BaseModel):
         list[tuple[tractor.MsgStream, Optional[float]]]
     ] = {}
 
-    class Config:
-        arbitrary_types_allowed = True
-        underscore_attrs_are_private = False
+    async def start_task(
+        self,
+        target: Awaitable,
+        *args,
+
+    ) -> None:
+
+        async def start_with_cs(
+            task_status: TaskStatus[
+                trio.CancelScope] = trio.TASK_STATUS_IGNORED,
+        ) -> None:
+            with trio.CancelScope() as cs:
+                self.nursery.start_soon(
```

Comment:
I think we should probably move this to be a `self.nusery.start()` for now until https://github.com/python-trio/trio/issues/2258 is resolved.